app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -183,6 +183,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
         } else if (i == R.id.action_delete_calendar) {
             deleteCalendarFromGoogle();
             return true;
+        } else if (i == R.id.action_create_event){
+            //new CreateEventDialogFragment().show(getSupportFragmentManager(), "CreateEventDialog");
```

Comment:
please don't leave commented code

Reply:
@pfent I know, could you please ignore this pull request for now, I know there are lots off errors and stuff in here right now. I just haven't pushed the next commit yet.

Reply:
sure, no worriez.
didn't notice the WIP label, my bad